Frameworks/UIKit/UIWindow.mm

```diff
@@ -360,10 +319,20 @@ - (UIWindowLevel)windowLevel {
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
+ @Notes Multi-screen apps can be built using WinRT APIs.
 */
 - (void)setScreen:(UIScreen*)screen {
-    UNIMPLEMENTED();
+    UNIMPLEMENTED_WITH_MSG("Multi-screen apps can be built using WinRT APIs.");
```

Comment:
this doesn't crash; it just writes a message to the debugger....?

---
In reply to: [110295163](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r110295163) [](ancestors = 110295163)